context/app/static/js/components/Detail/SectionItem.jsx

```diff
@@ -10,10 +11,21 @@ function SectionItem(props) {
   const { children, label, ml } = props;
```

Comment:
```suggestion
  const { childrenArray } = Array.isArray(children) ? children : [children];
```
I  rather give ourselves more uniform data, so we don't need as much complicated logic inside the JSX.